tests/reporter/kubernetes.go

```diff
@@ -437,7 +437,7 @@ func (r *KubernetesReporter) logVirtLauncherPrivilegedCommands(virtCli kubecli.K
 		if virtHandlerPod, ok := nodeMap[virtLauncherPod.Spec.NodeName]; ok {
 			labels := virtLauncherPod.GetLabels()
 			if uid, ok := labels["kubevirt.io/created-by"]; ok {
-				pid, err := getVirtLauncherPID(virtCli, &virtHandlerPod, uid)
+				pid, err := getVirtLauncherMonitorPID(virtCli, &virtHandlerPod, uid)
 				if err != nil {
 					continue
```

Comment:
> can you tell in the PR text when was the change introduced? 

Will add

> If I understand correctly, it caused a regression here because we silently `continue` on error. Do we want to keep doing so?

We do want to continue on error, k8s reporter it's best effort,
if it can't access a virt-launcher, it will try the other ones
not just here but best effort all over in the reporter, for example in the commands themselves

> BTW, what about the `--no-fork` on `pkg/virt-handler/isolation/detector_test.go`? Is it still relevant?

Good question, also saw that, didn't go deeper yet on it. maybe the file doesn't run, will do a quick check

Reply:
https://github.com/kubevirt/kubevirt/pull/7902